src/components/src/ProductDisplayItem/productdisplayitem.main.tsx

```diff
@@ -502,6 +502,14 @@ class ProductDisplayItemMain extends Component<ProductDisplayItemMainProps, Prod
     this.setState({ detailsProductData });
   }
 
+  handleArLinkClick(e) {
+    e.currentTarget.addEventListener('message', (event) => {
+      if (event.data === '_apple_ar_quicklook_button_tapped') {
+        this.addToCart(event);
```

Comment:
Nice! Does this work for products that require configurations? Ex. If the product requires line item modifiers to be set, or some config, will this still fire?

My suggestion is to only show the "add to cart" view if the addtocartaction is available, and it's available. We can selectively pass the "calltoaction" only under the following conditions `{availability && productData._addtocartform}`. That should give us a clue to whether we can add the item to cart in this view.